README.md

```diff
@@ -2,6 +2,8 @@
 
 Ramsey Solutions Frontend configurations and standards.
 
+We also recommend using [Prettier](https://prettier.io/) for formatting code in frontend repositories, but we use the default settings so there's no config file needed.
```

Comment:
🗣 we do actually use `"singleQuote": true` and `"arrowParens": "always"`, but I'm not sure if we have that configured here.

Reply:
@joshwhatk there's more of a conversation around this happening here: https://github.com/lampo/dev-standards/pull/507

Reply:
Thanks, @sauderrs!